recbot/app/lib/attestation-matcher.ts

```diff
@@ -68,7 +78,9 @@ export const isValidSkill = (text: string, mentioned_profiles: User[]) => {
     .map((skill) => skill.name)
     .includes(skill);
   const validUsername = mentionedUsernames.includes(username);
-  const isValid = match && validSkill && validUsername;
+  const icebreakerUser = await getIcebreakerUserFromFCUser(username) as IcebreakerUser
```

Comment:
We only need to do this lookup if it's a recursive attestation. If it's open, anyone can do it. 

Human and qBuilder are recursive, hence the check. Take a look at alloy for details

Reply:
Ah gotcha, makes sense!